### ISSUE TITLE
Last access to model in profile page

### ISSUE THREAD

**Issue by ClaraLavina** [juju/juju-gui#2610]
Should last access to your models in profile page say "Now" if you have a model open in a different tab?

<img width="1269" alt="screen shot 2017-03-08 at 12 30 37" src="https://cloud.githubusercontent.com/assets/22299251/23717948/72de792a-03fb-11e7-9b86-35fdb391ee72.png">

**Comment by frankban** [comment]
Yes it should, but we are not updating that page real time. Good catch thanks!
